packages/ruby/README.md

```diff
@@ -25,73 +25,71 @@ from the environment, or you may hardcode them.
+`allow_only`        | Array of strings | The inverse of `reject_params`. If included all parameters but those in this list will be redacted. NOTE: cannot be used in conjunction with `reject_params`. You may only specify either `reject_params` or `allow_only` keys, not both.
+`development`      | bool             | Defaults to `false`. When `true`, the log will be marked as a development log. This is great for separating staging or test data from data coming from customers.
+`buffer_length`     | number           | Defaults to `1`. This value should be a number representing the amount of requests to group up before sending them over the network. Increasing this value may increase performance by batching, but will also delay the time until logs show up in the dashboard given the buffer size needs to be reached in order for the logs to be sent.
 
-### Sensitive Data
+### Payload Data
 
-If you have sensitive data you'd like to prevent from being sent to the Metrics
-API via headers, query params or payload bodies, you can specify a list of keys
-to filter via the `reject_params` option. Key-value pairs matching these keys
-will not be included in the request to the Metrics API.
-
-You are also able to specify a set of `allow_only` which should only be sent through.
-Any header or body values not matching these keys will be filtered out and not
-send to the API.
-
-You may only specify either `reject_params` or `allow_only` keys, not both.
+Option              | Required? | Type             | Description
+--------------------|-----------|------------------|----------
+`api_key`           | yes       | string           | API Key used to make the request. Note that this is different from the `readmeAPIKey` described above in the options data. This should be a value from your API that is unique to each of your users.
+`label`             | no        | string           | This will be the user's display name in the API Metrics Dashboard, since it's much easier to remember a name than an API key.
+`email`             | no        | string           | Email of the user that is making the call.
+`log_id`            | no        | string           | A UUIDv4 identifier. If not provided this will be automatically generated for you. Providing your own `log_id` is useful if you want to know the URL of the log in advance, i.e. `{your_base_url}/logs/{your_log_id}`.
 
 ### Rails
 
 ```ruby
 # config/environments/development.rb or config/environments/production.rb
 require "readme/metrics"
 
-options = {
+sdk_options = {
   api_key: "<<apiKey>>",
   development: false,
   reject_params: ["not_included", "dont_send"],
   buffer_length: 5,
 }
 
-config.middleware.use Readme::Metrics, options do |env|
+config.middleware.use Readme::Metrics, sdk_options do |env|
   current_user = env['warden'].authenticate
 
-  if current_user.present?
-    {
```

Comment:
Is/was this Ruby shorthand for just returning an object?

Reply:
Yeah I wanted to name the variable to make the docs more clear in terms of what this is referring to